### ISSUE TITLE
autoload network.gd script functions wont fire when called from another node

### ISSUE THREAD

**Issue by mojoyup** [godotengine/godot#66324]
### Godot version

4.0 beta1 20d667284

### System information

Windows 10 64, GLES3, NVIDIA 1660Ti dedicated

### Issue description

I have a network script as an autoload. I have a scene with another script that has a button that is to create_server(). The button does not call any function from the network.gd script though it can from another script. I also have a firebase.gd autoload that does get called. SO, I don't believe its an autoload issue. 

It works in the editor but on export is when it doesn't work on the network.gd. Firebase autoload still fires. 
I have exported with/without pck attached, with/without debug, have tried call(), call_deferred(), direct script call "network.'thefunction()', etc. 

If I make any function from another script, the other script functions are called fine. Ive noted one such example commented out in the func _on_host_btn+pressed() in the mp_panel.gd script(Script for the muliplayer panel that has the two buttons to host and join). 

I hope that's enough information for now. 
I can make a video recording of the process if needed and share. YOU SHOULD BE ABLE TO LOG IN WITH THE PROVIDED INFORMATION IN THE LOG IN FIELDS

Im attaching my proj file. Please do not replicate it, redistribute it, resell it, or use it in any fashion other than for the purposes of this debugging issue. 
[multiplayer_template.zip](https://github.com/godotengine/godot/files/9637214/multiplayer_template.zip)

NOTE: I also just noticed that rcedit is giving me an error saying 'Transient parent has another exclusive child. I tried the parent folder also. 

### Steps to reproduce

UNZIP the proj and run. The button to host does not fire the function host_server() in the Network autoload. 
Login,  then try to host. It is suppoesed to spawn the player on a plane mesh as a box. You will see the player and the ground plane if it is successful and/or a message label top center as well stating "I am added to the server!"

**Comment by Calinou** [comment]
@mojoyup Did you find a solution to this problem? If so, please document it here by commenting :slightly_smiling_face:

**Comment by mojoyup** [comment]
I did yes, I studied the code for a while before reporting this to make sure I didn't have to create a workload not necessary, but couldn't until after I posted :P So, basically, it was a Parse error from the exported cmd generated with the exe. It say that a preload was not recognizing the scene. I fixed it by making the preload a load and retested the export both exe and cmd. They both worked :)

My apologies for the confusion :P I tried to find that out beforehand.
